seamare/src/content.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO: it's just copied from x-linter, maybe we don't need this `ContentLinter` and `ContentContext`?
```

Comment:
```suggestion
```

I think we will want this eventually, the plan is to make `cargo seamare` and friends generally available to the rust community, so I expect further lints will get added in the future.